Makefile

```diff
@@ -145,6 +145,29 @@ scoop: all
 	@ echo "Running minimal example with SCOOP"
 	python -m scoop -n 2 rmg.py -v testing/scoop/input.py
 
+.ONESHELL:
+update:
+	@ echo "Updating RMG-py"
+	@ echo "Please remember to stash unwanted changes."
+	git status
+	@ read -p "Are there any changes that are not staged for committed? [y|n] " user_input
+	if [ "$$user_input" = "N" ] || [ "$$user_input" = "n" ]; then
```

Comment:
does this work on your system? because I have the following error when I select y or n: 
```
(rmg_env) blais.ch@FACC02D75Y4MD6T RMG-Py % make update
Updating RMG-py
Please remember to stash unwanted changes.
git status
On branch make_update
Your branch is up to date with 'origin/make_update'.

nothing to commit, working tree clean
Are there any changes that are not staged for committed? [y|n] y
if [ "$user_input" = "N" ] || [ "$user_input" = "n" ]; then
/bin/sh: -c: line 1: syntax error: unexpected end of file
make: *** [update] Error 2
```
this message appears in both zsh and bash. I assume it has to do with mixing make and bash?